### DISCUSSION TITLE
CloudWatch vs CloudWatchLogs

### DISCUSSION THREAD

**Discussion by rvdbrandatn** [aws/aws-sdk-net#3063] [General]
What is the difference between these two when the have the same description? The version numbers are different, so there must be differences. But where? and more, why?

**Comment by ashovlin** [comment]
1. CloudWatch corresponds to [this set of APIs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/Welcome.html), which include metrics, alarms, dashboards, etc.
2. CloudWatch Logs is [this set of APIs](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/Welcome.html), which covers log events, groups, streams, etc.

----

It looks like we do have the package description incorrectly duplicated from CloudWatch to CloudWatch Logs here: https://github.com/aws/aws-sdk-net/blob/e2ae36801fe92bee601a273d09ac4f1f60daf57f/generator/ServiceModels/logs/metadata.json#L3C1

Which should be updated to match: https://github.com/aws/aws-sdk-net/blob/e2ae36801fe92bee601a273d09ac4f1f60daf57f/generator/ServiceModels/logs/logs-2014-03-28.docs.json#L3

Thanks for pointing that out.
